### ISSUE TITLE
Proxy?

### ISSUE THREAD

**Issue by ThiloL** [aaronparker/LatestUpdate#16]
Please include proxy support for thid module.

**Comment by aaronparker** [comment]
Proxy support has been added into `Save-LatestUpdate` in #40. I'll add proxy support for the other functions as soon as I can. I don't really have a proxy to test against though, so testing on the module is needed.

**Comment by f0nt4** [comment]
I've updated my fork for proxy support in all the public functions. Not created a PR yet because I still need to test it.

**Comment by aaronparker** [comment]
I think proxy support can be added externally to the function. I'll research

**Comment by f0nt4** [comment]
Yeah, I was thinking the same. Otherwise it will be a lot of code.
